database/schema.sql

```diff
@@ -5,3 +5,24 @@ set client_min_messages to warning;
 drop schema "public" cascade;
 
 create schema "public";
+
+CREATE TABLE "public"."restaurants" (
+	"id"          serial   NOT NULL,
```

Comment:
I thought I had suggested you name this "restaurantId"? That will make your joins easier.